nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -61,12 +112,78 @@ void TfOpNode::setLayerNode(const LayerNode &layer) {
 
 void TfOpNode::setWeightTransformFn(TransformFn fn) { weight_transform = fn; }
 
+void TfOpNode::setInputTransformFn(TransformFn fn) { input_transform = fn; }
+
+void TfOpNode::weightReorder(unsigned int node_count) {
+
+  if (need_reorder_weight == true) {
+
+    auto previous_input_shape = input_nodes[0]->getInputs()[0];
+
+    const unsigned int UNIT = outputs[0]->height();
+    const unsigned int CHANNEL = previous_input_shape->channel();
+    const unsigned int HEIGHT = previous_input_shape->height();
+    const unsigned int WIDTH = previous_input_shape->width();
+
+    auto weight_data = weights[0]->getData();
+    auto *ptr = const_cast<float *>(weight_data);
+
+    std::vector<float> old_value_list;
+    old_value_list.reserve((UNIT * CHANNEL * HEIGHT * WIDTH));
```

Comment:
```suggestion
    std::vector<flaot> old_value_list(UNIT * CHANNEL * HEIGHT * WIDTH);
```

Is there any type for `std::vector<flaot>` ?